timber/build.gradle

```diff
@@ -1,5 +1,7 @@
 apply plugin: 'com.android.library'
 apply plugin: 'org.jetbrains.kotlin.android'
+apply plugin: 'org.jetbrains.dokka'
```

Comment:
not required?

Reply:
I'm assuming from Jake's comment [here](https://github.com/vanniktech/gradle-maven-publish-plugin/issues/211) that it isn't required, so I'll update and remove.